SumLists.py

```diff
@@ -0,0 +1,39 @@
+class ListNode:
+    def __init__(self, value=0, next=None):
+        self.value = value
+        self.next = next
+
+def addNums(l1, l2):
+    head = ListNode()
+    cur = head
+    carry = 0
+    while l1 or l2 or carry:
+        sum_val = carry
```

Comment:
Is there a reason you didn't just call this `sum`? While there isn't anything technically incorrect with naming it `sum_val`, appending more to variable names for unclear reasons can make your names more unpredictable, which can make your code less intuitive to other programmers.